### DISCUSSION TITLE
Tasmota zigbee bridge, channel change. How to do?

### DISCUSSION THREAD

**Discussion by Hoeby** [arendst/Tasmota#13765] [Q&A] [answered]
I am using a sonoff zigbee bridge, flashed with tasmota. Works great, thanks to all for the hard work.

But i am searching for channel change, but can't find exactly how to.

In the configuration.h is #define USE_ZIGBEE_CHANNEL 11
Could i make a user override config, change the number (between 11 and 26) and compile it and flash the bridge with the compiled version. Will the zigbee channel then be changed?

I am not working with homeassistant, can't do it from a plugin or something. Need to do it in the code itself.

**Comment by ascillato** [comment, marked as answer]
There is no need to recompile. If you recompile, you must do a `reset 6` and then a `zbreset 1` in order to change to those new default values.

You can just use in the Tasmota console the commands of the docs ( https://tasmota.github.io/docs/Commands/#zbconfig ):

`ZbScan` to do an energy scan on each radio channel

`ZbConfig <json>` to set parameters of the ZigBee Bridge

For changing the channel, just type in the console:

```lua
ZbConfig {"Channel":12}
```

Be aware that you may need to re-pair all your devices.
